client/src/components/Forms/index.tsx

```diff
@@ -1,9 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { FirmwareState, Input, Output, ServerInput } from './Input_Types';
+import InputForm from './InputForm';
+import OutputForm from './OutputForm';
+import ServerForm from './ServerForm';
+import SummaryForm from './SummaryForm';
+import MenuSwitcher from './MenuSwitcher';
 
-const Form: React.FunctionComponent = () => (
-  <div className="container">
-    <p>Form adding soon</p>
-  </div>
-);
+const initialFirmwareState: FirmwareState = {
+  server: {} as ServerInput,
+  inputs: [] as Input[],
+  outputs: [] as Output[],
+};
 
-export default Form;
+/**
+ * @returns Top level functional component for input, output and server form pages
+ */
+const Forms: React.FunctionComponent = () => {
+  const [state, setState] = useState<FirmwareState>(initialFirmwareState);
+  const [form, setForm] = useState('InputForm');
+
+  /**
+   * Handler to append input to list from create input modal
+   * @param userInputs created input
+   */
+  const handleInputChange = (userInputs: Input): void => {
+    setState({ ...state, inputs: state.inputs.concat(userInputs) });
+  };
+
+  /**
```

Comment:
These are stub functions for when the output component and server component has been built. I made them as I was writing the state model for the application. Eventually, these functions will be passed down as props to their components so that form inputs in the components can make changes to this global state in Forms/index.tsx